<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Servicio</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="flight_button">
         <property name="text">
          <string>Vuelo</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="freight_button">
         <property name="text">
          <string>Encomienda</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>ej. John Doe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>RUT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="identification">
       <property name="placeholderText">
        <string>ej. 12.345.678-5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Destino</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="destination"/>
     </item>
     <item>
      <widget class="QLabel" name="airplane_label">
       <property name="text">
        <string>Avi√≥n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="airplane"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="date_label">
           <property name="text">
            <string>Fecha de ida</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCalendarWidget" name="date"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="time_label">
           <property name="text">
            <string>Hora de ida</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="time"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="seats_label">
       <property name="text">
        <string>Asientos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="seats">
       <property name="placeholderText">
        <string>ej. 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="weight_label">
       <property name="text">
        <string>Peso (en kilogramos)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weight">
       <property name="placeholderText">
        <string>ej. 10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Medio de pago</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="payment_method"/>
     </item>
     <item>
      <widget class="QPushButton" name="nullify_payment_button">
       <property name="text">
        <string>Obviar pago</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
